example-ts/nft.ts

```diff
@@ -0,0 +1,147 @@
+import * as THREE from 'three'
+import { THREEx, ARjs } from "@ar-js-org/ar.js-threejs"
+
+THREEx.ArToolkitContext.baseURL = "./";
+// log in the console the two newly created namespaces
+console.log(THREEx);
+console.log(ARjs)
+//////////////////////////////////////////////////////////////////////////////////
+//		Init
+//////////////////////////////////////////////////////////////////////////////////
+
+// init renderer
+var renderer = new THREE.WebGLRenderer({
+    antialias: true,
+    alpha: true
+});
+renderer.setClearColor(new THREE.Color('lightgrey'), 0)
+renderer.setSize(640, 480);
+renderer.domElement.style.position = 'absolute'
+renderer.domElement.style.top = '0px'
+renderer.domElement.style.left = '0px'
+document.body.appendChild(renderer.domElement);
+
+// array of functions for the rendering loop
+var onRenderFcts: any[] = [];
+var arToolkitContext: any, arMarkerControls;
+
+// init scene and camera
+var scene = new THREE.Scene();
+
+//////////////////////////////////////////////////////////////////////////////////
+//		Initialize a basic camera
+//////////////////////////////////////////////////////////////////////////////////
+
+// Create a camera
+var camera = new THREE.PerspectiveCamera();
+scene.add(camera);
+
+
+var arToolkitSource = new THREEx.ArToolkitSource({
+    // to read from the webcam
+    sourceType: 'webcam',
+
+    sourceWidth: window.innerWidth > window.innerHeight ? 640 : 480,
+    sourceHeight: window.innerWidth > window.innerHeight ? 480 : 640,
+})
+
+arToolkitSource.init(function onReady(){
+    // use a resize to fullscreen mobile devices
+    setTimeout(function() {
+        onResize()
+    }, 1000);
+}, function onError() { })
+
+// handle resize
+window.addEventListener('resize', function(){
+    onResize()
+})
+
+// listener for end loading of NFT marker
+window.addEventListener('arjs-nft-loaded', function(ev){
+    console.log(ev);
+})
+
+function onResize(){
+    arToolkitSource.onResizeElement()
+    arToolkitSource.copyElementSizeTo(renderer.domElement)
+    if( arToolkitContext.arController !== null ){
+        arToolkitSource.copyElementSizeTo(arToolkitContext.arController.canvas)
+    }
+}
+
+////////////////////////////////////////////////////////////////////////////////
+//          initialize arToolkitContext
+////////////////////////////////////////////////////////////////////////////////
+
+// create atToolkitContext
+var arToolkitContext: any = new THREEx.ArToolkitContext({
+    detectionMode: 'mono',
+    cameraParametersUrl: 'https://ar-js-org.github.io/AR.js/data/data/camera_para.dat',
```

Comment:
i give a https path to **cameraParamtersUrl** because it was hard to give the right path for the arController in the Worker and that outside in the atTooolkitContext. **cameraParamtersUrl** is not optional so you must provide a path, in the old implementation it will provide a defaulth path if not provided. I will fix this.